projects/hslayers/src/components/add-data/url/geosparql/geosparql.service.ts

```diff
@@ -0,0 +1,88 @@
+import {HttpClient} from '@angular/common/http';
+import {Injectable} from '@angular/core';
+import {lastValueFrom, takeUntil} from 'rxjs';
+
+import {HsAddDataService} from '../../add-data.service';
+import {HsAddDataUrlService} from '../add-data-url.service';
+import {HsLanguageService} from '../../../language/language.service';
+import {HsLogService} from '../../../../common/log/log.service';
+import {urlDataObject} from '../types/data-object.type';
+
+class HsUrlGeoSparqlParams {
+  data: urlDataObject;
+
+  constructor() {
+    this.data = {
+      table: {
+        trackBy: 'id',
+        nameProperty: 'name',
+      },
+    };
+  }
+}
+
+@Injectable({providedIn: 'root'})
+export class HsUrlGeoSparqlService {
+  apps: {
+    [id: string]: any;
+  } = {default: new HsUrlGeoSparqlParams()};
+
+  constructor(
+    public httpClient: HttpClient,
+    public hsAddDataService: HsAddDataService,
+    public hsLanguageService: HsLanguageService,
+    public hsLog: HsLogService,
+    public hsAddDataUrlService: HsAddDataUrlService
+  ) {}
+
+  get(app: string): HsUrlGeoSparqlParams {
+    if (this.apps[app ?? 'default'] === undefined) {
+      this.apps[app ?? 'default'] = new HsUrlGeoSparqlParams();
+    }
+    return this.apps[app ?? 'default'];
+  }
+
+  async verifyEndpoint(url: string, app: string = 'default') {
+    try {
+      const r = await lastValueFrom(
+        this.httpClient
+          .get(url, {
+            //TODO: could there be also different accept header?
+            headers: {'Accept': ['application/rdf+xml', 'application/xml']},
+            responseType: 'blob',
+          })
+          .pipe(takeUntil(this.hsAddDataService.cancelUrlRequest))
+      );
+      const blobText = await r.text();
+      const parsedResponse = new DOMParser().parseFromString(
+        blobText,
+        'application/xml'
+      );
+      if (parsedResponse.activeElement.localName === 'RDF') {
```

Comment:
Ignoring the letter case might be safer

Reply:
I had to read your comment at least 10 times, until I realised the difference between "the **letter** case" and "the **latter** case" 🧐 
Now it makes sense and it is a good suggestion indeed!

Reply:
Fixed in fbbd5ab (#3310)